### ISSUE TITLE
Buttons' collours enhancement

### ISSUE THREAD

**Issue by tristone-cz** [nervetattoo/simple-thermostat#92]
Hi,
Would it be possible to rely for the button background colours to the predefined colours of the theme?

Now some colours are specified strictly by the hexadeciaml values and does not fit to other themes then the default one.

Not critical, just an improvement suggetion.

Thanks.

**Comment by tristone-cz** [comment]
I made some small research and I would suggest just to change two default values to utilitize the variables which should be present in any theme.
The ability to skin via card-mod or theme specific variables will not be influenced this way.

.mode-item {
    background: var(--st-mode-background, var(**--secondary-background-color**));

.mode-item:hover {
    background: var(--st-mode-active-background, var(**--light-primary-color**));

Tested with default theme and themes clear / clear-dark and it seems to work as intended.

**Comment by nervetattoo** [comment]
Thanks for the suggestion.
This makes sense, but it didn't look great with default theme with those exact values so I'll change the defaults but to use

This for inactive
```css
.mode-item {
    background: var(--st-mode-background, var(--secondary-background-color));
    color: var(--secondary-text-color);
}
.mode-item:hover {
    color: var(--st-mode-active-color, var(--primary-text-color));
}
.mode-item.active, .mode-item.active:hover {
    background: var(--st-mode-active-background, var(--primary-color));
    color: var(--st-mode-active-color, var(--text-primary-color));
}
```

Which will look like this for default theme:
<img width="486" alt="Screenshot 2020-01-31 at 23 03 29" src="https://user-images.githubusercontent.com/45449/73577731-fcbbfa80-447d-11ea-9a23-782ac24b9589.png">

How will that work out for your use case?

**Comment by tristone-cz** [comment]
Hi.

It seesm to work quite fine :+1: 

I just assume that in `mode-item:hover` you removed `background` setting and keep just `color`.
If `backgroud` is kept as now in the clear-dark theme text and background are the same for hover over inactive.

In bellow picure is the day and night (themes clear and clear-dark) default and hover. Almost no difference.
![image](https://user-images.githubusercontent.com/40542489/73595446-3857d280-4519-11ea-9f51-34d9128922f9.png)

**Comment by nervetattoo** [comment]
Released in 0.34.

**Comment by tomachi** [comment]
> Hi.
> 
> It seesm to work quite fine 👍
> 
> I just assume that in `mode-item:hover` you removed `background` setting and keep just `color`.
> If `backgroud` is kept as now in the clear-dark theme text and background are the same for hover over inactive.
> 
> In bellow picure is the day and night (themes clear and clear-dark) default and hover. Almost no difference.
> ![image](https://user-images.githubusercontent.com/40542489/73595446-3857d280-4519-11ea-9f51-34d9128922f9.png)
> 
> I am personally using `card-mod` to update the icon color to match better my dashboard, but that's different story 😉
> ![image](https://user-images.githubusercontent.com/40542489/73595529-08f59580-451a-11ea-83ad-b07776fb5efc.png)

Can you explain how you did it?
